src/common/models/Projection.ts

```diff
@@ -303,6 +305,25 @@ export class Projection {
     return getRegionMetricOverride(this.region, metric)?.blocked ?? false;
   }
 
+  get twoWeekPercentChangeInCases() {
```

Comment:
This "two week percentage" logic moved from NationalText to here so we could store it in the site-summary.json file.